src/modules/person/api/person-api.mapper.profile.ts

```diff
@@ -82,6 +83,35 @@ export class PersonApiMapperProfile extends AutomapperProfile {
                 ),
             );
             createMap(mapper, CreatePersonDto, PersonDo);
+            createMap(
+                mapper,
+                PersonDo,
+                PersonResponse,
+                forMember(
+                    (dest: PersonResponse) => dest.id,
+                    mapFrom((src: PersonDo<true>) => src.id),
+                ),
```

Comment:
we need an explicit mapper rule for the id because id has a different data type in PersonRespone then in PersonDo. The mapping without this rule caused errors in tests.